hashes/Cargo.toml

```diff
@@ -30,7 +30,7 @@ core2 = { version = "0.3.0", default_features = false, optional = true }
 # Only enable this if you explicitly do not want to use "std", otherwise enable "serde-std".
 serde = { version = "1.0", default-features = false, optional = true }
 # Do NOT use this as a feature! Use the `schemars` feature instead. Can only be used with "std" enabled.
-actual-schemars = { package = "schemars", version = "<=0.8.3", optional = true }
+actual-schemars = { package = "schemars", version = "0.8.12", optional = true }
```

Comment:
Why not keep it at 0.8.3 if we don't need the features in 0.8.12?

Reply:
This should not have been in the PR, my bad.